uploader/src/main/kotlin/com/cloudinary/upload/ApiCalls.kt

```diff
@@ -87,6 +87,19 @@ fun Uploader.addTag(
     publicIds: List<String>,
     exclusive: Boolean = false,
     request: (TagsRequest.Builder.() -> Unit)? = null
+): UploaderResponse<TagsResult> {
+    val command = if (exclusive) TagsCommand.SetExclusive else TagsCommand.Add
```

Comment:
Here you can call the function with the list of tags defined below. No need to write implementation twice.